src/core/public/chrome/ui/header/header_badge.tsx

```diff
@@ -66,7 +66,7 @@ export class HeaderBadge extends Component<Props, State> {
       <div className="chrHeaderBadge__wrapper">
         <EuiBetaBadge
           data-test-subj="headerBadge"
-          data-test-badge-label={this.state.badge.text}
+          data-test-badge-label={(this.state.badge.text as unknown) as undefined}
```

Comment:
A hack to mute the error: `Types of property 'data-test-badge-label' are incompatible. Type 'string' is not assignable to type 'undefined'.` We should extend the list of common props in `eui` with the known attributes if I'm not mistaken.
@thompsongl @chandlerprall

Reply:
This is a bug introduced by the stricter index signature checks introduced in 3.8. https://github.com/microsoft/TypeScript/issues/36935

The resolution was milestoned for 3.9, but there appears to be some interest to include a fix in 3.8

Reply:
the fix was backported to 3.8. waiting for the next patch release 👍

Reply:
updated to 3.8.3